youtube_dl/extractor/aparat.py

```diff
@@ -11,10 +11,10 @@
 
 
 class AparatIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?aparat\.com/(?:v/|video/video/embed/videohash/)(?P<id>[a-zA-Z0-9]+)'
+    _VALID_URL = r'https?://(?:www\.)?aparat\.com/(?:v/|video/video/embed/videohash/)(?P<id>[a-zA-Z0-9]+/|vt/frame)'
```

Comment:
Shouldn't that slash be optional at least?

Reply:
yes, you right...I tried that but it won't work and will get the regex error again, I don't know why it happens🤔.
surprisingly putting slash is still optional, for users😅(the below command works!)
```bash youtube-dl  http://www.aparat.com/v/wP8On ```